training/src/pages/InputDemo/InputDemo.jsx

```diff
@@ -13,11 +13,8 @@ import {
 class InputDemo extends React.Component {
    schema = yup.object().shape({
      name: yup.string().required('Please Enter your Name').min(3, 'Please enter no less than 3 characters'),
-     sport: yup.string().required('Sport is required field '),
-     cricket: yup.string().when('sport', {
-       is: 'cricket',
-       then: yup.string().required('What you do is a required field'),
-     }),
+     email: yup.string().required('Please Enter your Em').min(3, 'Please enter no less than 3 characters'),
+     password: yup.string().required('Sport is required field '),
```

Comment:
The text should be `Password is required field` instead of `Sport is required field `